LoopKit.xcodeproj/project.pbxproj

```diff
@@ -1929,13 +2061,13 @@
 				43D9888C1C87EBE400DA4467 /* LoopMathTests.swift */,
 				43D8FE1B1C72906E0073BE78 /* NSDateTests.swift */,
 				A95A1D882460F8930079378D /* PumpManagerStatusTests.swift */,
-				A99C7372233990D400C80963 /* TempBasalRecommendationTests.swift */,
-				8974AFBF22120D7A0043F01B /* TemporaryScheduleOverrideTests.swift */,
-				891A3FD02249948A00378B27 /* TemporaryScheduleOverrideHistoryTests.swift */,
 				434C5F9F209ABD4700B2FD1A /* QuantityFormatterTests.swift */,
 				43D8FE1C1C72906E0073BE78 /* QuantityScheduleTests.swift */,
-				A971C8A123C6B17D0099BEFC /* SettingsStoreTests.swift */,
 				A99C73782339ACDC00C80963 /* ServiceTests.swift */,
+				A971C8A123C6B17D0099BEFC /* SettingsStoreTests.swift */,
+				A99C7372233990D400C80963 /* TempBasalRecommendationTests.swift */,
```

Comment:
[nit] What is all of this? It doesn't seem relevant to this PR.

Reply:
This is the result of creating a new target